app/models/maestrano/connector/rails/concerns/entity.rb

```diff
@@ -234,7 +234,7 @@ def get_connec_entities(last_synchronization_date = nil)
       # Fetch subsequent pages
       while response_hash['pagination'] && response_hash['pagination']['next']
         # ugly way to convert https://api-connec/api/v2/group_id/organizations?next_page_params to /organizations?next_page_params
-        next_page = response_hash['pagination']['next'].gsub(/^(.*)\/#{self.class.normalized_connec_entity_name}/, self.class.normalized_connec_entity_name)
+        next_page = response_hash['pagination']['next'].gsub(/\A(.*)\/#{self.class.normalized_connec_entity_name}/, self.class.normalized_connec_entity_name)
```

Comment:
`\A` is correct. Capture group looks unnecessary, could be `/\A.*\/...` rather than `/\A(.*)\/...`. Other than that LGTM.